### PR TITLE
Impress: add minimal Curve inserting

### PR DESCRIPTION
**Pull request by meven**
Change-Id: Ie10752545df5eb3b5baefab22c583bfa72611244

### Summary

Add minimal Curve inserting
 
image

UI is not present when drawing

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by vmiklos** [comment]
https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04_cypress_desktop/1795/console failed in:

> writer/invalidations_spec.js
> writer/table_operation_spec.js

Retry this as well.

File: browser/src/unocommands.js [hunk 1]
```diff
@@ -251,6 +252,7 @@ var unoCommandsArray = {
 	'IncrementLevel':{text:{menu:_('Promote Outline Level'),},},
 	'IncrementSubLevels':{text:{menu:_('Promote Outline Level with Subpoints'),},},
 	'IndexesMenu':{text:{menu:_('Table of Contents and Inde~x'),},},
+	'InsertAVMedia':{global:{context:_('Insert Audio or Video'),menu:_('Media'),},},
```
Comment (hfiguiere) [inline browser/src/unocommands.js:255]: this change should be removed like its counterpart.

Comment (meven) [reply browser/src/unocommands.js:255]: Fair.
The script ought to run again, in a different PR.

Comment (hfiguiere) [reply browser/src/unocommands.js]: we can fix that in a different PR. 

I wonder why this is out of order....

**Comment by vmiklos** [comment]
https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04/2624/console fails in:

> unit-save-torture

retry.

File: browser/src/control/Control.NotebookbarImpress.js [hunk 2]
```diff
@@ -1461,6 +1461,12 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 					}
 				],
 				'vertical': 'true'
+			},
+			{
```
Comment (vmiklos) [inline browser/src/control/Control.NotebookbarImpress.js:1465]: My notebookbar now looks like this:

image

Is it expected that ~all other buttons have a label next to the icon, but not this one? I would expect the new button would look similar to the existing line button, or something like that. Thanks.

Comment (meven) [reply browser/src/control/Control.NotebookbarImpress.js:1465]: This was in my screenshot too.
I noticed, I am looking at it. I don't get why it doesn't show up.

Comment (hfiguiere) [reply browser/src/control/Control.NotebookbarImpress.js:1465]: also it's not in Draw....

Comment (meven) [reply browser/src/control/Control.NotebookbarImpress.js:1465]: Now fixed

**Comment by vmiklos** [comment]
Rebase to be on top of online.git commit 7998ba53a533be83ce8fed60cabc44b6457543c4

**Review by vmiklos — APPROVED** [review]
The actual behavior looks a bit confusing to me (even on the desktop), but this does expose the feature in COOL anyway, so look OK to me.

**Pull request merged**